cleverhans/experimental/certification/nn.py

```diff
@@ -11,23 +11,21 @@
 import numpy as np
 import tensorflow as tf
 
-STRIDE = 2
-PADDING = 'SAME'
-
 class NeuralNetwork(object):
   """NeuralNetwork is a class that interfaces the verification code with
   the neural net parameters (weights).
   """
 
-  def __init__(self, net_weights, net_biases, net_layer_types, input_shape=None):
+  def __init__(self, net_weights, net_biases, net_layer_types, input_shape=None, cnn_params=None):
     """Function to initialize NeuralNetParams class.
 
     Args:
       net_weights: list of numpy matrices of weights of each layer
        [convention: x[i+1] = W[i] x[i]
       net_biases: list of numpy arrays of biases of each layer
       net_layer_types: type of each layer ['ff' or 'ff_relu' or 'ff_conv' or
-        'ff_conv_relu']
+        'ff_conv_relu']      vector = tf.reshape(vector, self.input_shapes[layer_index])
```

Comment:
Is `vector = tf.reshape(vector, self.input_shapes[layer_index])` added accidentally?